facia-press/app/controllers/HealthCheck.scala

```diff
@@ -11,13 +12,17 @@ class HealthCheck(
   toolPressQueueWorker: ToolPressQueueWorker,
   val controllerComponents: ControllerComponents
 )
-  extends HealthCheckController with Results {
+  extends HealthCheckController with Results with Logging {
   val ConsecutiveProcessingErrorsThreshold = 5
   override def healthCheck(): Action[AnyContent] = Action.async{
     if (!toolPressQueueWorker.lastReceipt.exists(_.plusMinutes(1).isAfter(DateTime.now))) {
-      successful(Results.InternalServerError("Have not been able to retrieve a message from the tool queue for at least a minute"))
+      val msg = "Have not been able to retrieve a message from the tool queue for at least a minute"
+      log.error(msg)
```

Comment:
I'm going to make the assumption here that `common.Logging` is an interface to ES that adds some meta data around the message? Sorry, just wanted to be sure I understand things right

Reply:
That's correct -- we can see this pattern in other controllers, as well, e.g. `MatchController`.

Reply:
Or, rather -- `adds some meta data around the message` -- I guess you can use it for that, but here I'm just following a convention used in other controllers which saves us writing `lazy implicit val log = Logger(getClass)`.